skizzophrenic/SquachWare-CFW
# Added NameChanger app and Service.

# What's new

- NameChanger App and Service. Taken from [ESurge's NameChanger](https://github.com/ESurge/flipperzero-namechanger)

# Verification 

- Run NameChanger app
- Change name
- Device will restart
- Check Passport for new name

- Name is stored in `SD Card/dolphin/name.txt` file

# Checklist (For Reviewer)
